app/controllers/repairs_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RepairsController < ApplicationController
+    def index
+        @repair = Repair.all 
+    end
+    def new
+        @repair = Repair.new
+        
+    end
```

Comment:
To follow the style guidelines, remove these blank lines inside of the function definition.